travis/test_module

```diff
@@ -0,0 +1,341 @@
+                self.logger.info('Checking module: ' + module)
+                self.check_description(module, filenames)
+                self.check_icon_missing(module)
+                self.check_doc(module)
+        self.log_error_summary()
+        return True
+
+    def log_error_summary(self):
+        """
+        log error summary with all kind of errors.
+        """
+        if self.desc_legacy:
+            self.logger.warn('='*80)
+            self.logger.warn('Warning Summary:')
+            self._log_warn(
+                self.desc_legacy,
+                'Using description key in __openerp__.py (legacy mode):')
+        if (self.syntax_error or self.missing_rst or self.icon_missing
+                or self.doc_missing):
+            self.logger.error('='*80)
+            self.logger.error('Error Summary:')
+            if self.syntax_error:
+                self._log_error(self.syntax_error, 'README.rst syntax errors:')
+            if self.missing_rst:
+                self._log_error(
+                    self.missing_rst, 'Missing README.rst files:')
+            if self.icon_missing:
+                self._log_error(self.icon_missing, 'Missing icon.png:')
+            if self.doc_missing:
+                self._log_error(self.doc_missing, 'Missing doc/index.rst:')
+            exit(1)
+        return True
+
+    def _log_error(self, module_list, msg):
+        """
+        call error log.
+        @return True
+        """
+        self.logger.error(self._log_msg(module_list, msg))
+        return True
+
+    def _log_warn(self, module_list, msg):
+        """
+        call warn log.
+        @return True
+        """
+        self.logger.warn(self._log_msg(module_list, msg))
+        return True
+
+    def _log_msg(self, module_list, msg):
+        """
+        Return message to use in the log msg to use in the script.
+        """
+        res = str()
+        if module_list:
+            res = [msg] + module_list
+            res = '\n    -'.join(res)
+        return res
+
+    def check_description(self, module, filenames):
+        """
+        This method check the module description.
+
+        - that the README.rst file exist.
+        - that the README.rst file have a valid rst syntaxt
+        - that the openerp.py have not the description key (legacy mode).
+
+        @return True
+        """
+        self.logger.info(module + ': Checking description')
+        if 'README.rst' in filenames:
+            self.check_rst_syntax(module, 'README.rst')
+        else:
+            self.missing_rst.append(module)
+
+        with open(os.path.join(module, '__openerp__.py'), 'r') as fopenerp:
+            data = fopenerp.read()
+            data = eval(data)
+            if 'description' in data.keys():
+                self.logger.warn(
+                    'Legagy mode: Using description in __openerp__.py file')
+                self.desc_legacy.append(module)
+        return True
```

Comment:
If you will not use any return simple remove the return statement and it will return None.